install.sh

```diff
@@ -255,9 +286,13 @@ nvm_do_install() {
     fi
   fi
 
+  # Sourcing $PROFILE to take into account the new environment
+  source $NVM_PROFILE
```

Comment:
sorry, one more - this needs to be quoted, and it must use `.` since `source` isn't as portable